src/algebra/group/pi.lean

```diff
@@ -16,6 +16,7 @@ This file defines instances for group, monoid, semigroup and related structures
 -/
 
 universes u v w
+variables {ι α : Type*}
```

Comment:
I would argue that you should use `I` instead of `ι` for consistency with the rest of the file.

Reply:
Actually, `I` is pretty clearly used for dependent functions while `ι` is used for non-dependent ones. See lines 425-436.